ui_framework/doc_site/src/views/typography/typography_example.js

```diff
@@ -11,9 +11,9 @@ import {
   GuideText,
 } from '../../components';
 
-import Typography from './typography';
```

Comment:
What do you think of splitting this example into separate examples for "Title" and "Text"? The two components don't seem to really collaborate with one another now -- titling seems to be one concern, and "drop-in text" seems like another, different concern.